Cannot find module '@axelar-network/axelarjs-sdk'

```bash
 npm run send-token

> axelar-local-dev-sample@1.0.0 send-token
> node src/send-token.js

node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '@axelar-network/axelarjs-sdk'
Require stack:
- /Users/gus/Documents/axelar/git-repos/axelar-local-dev-sample/src/send-token.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/gus/Documents/axelar/git-repos/axelar-local-dev-sample/src/send-token.js:4:27)
    at Module._compile (node:internal/modules/cjs/loader:1099:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/gus/Documents/axelar/git-repos/axelar-local-dev-sample/src/send-token.js'
  ]
}

Node.js v17.8.0
```

I thought the local-dev-sample no longer has a dependency on axelarjs-sdk.

--- a/src/call-contract-with-token.js
+++ b/src/call-contract-with-token.js
@@ -9,37 +9,37 @@ const {
 
 // Create multiple recipients
 const recipientAddresses = generateWalletAddresses(5);
-const alias = recipientAddresses.map((_, i) => `Recipient${i + 1}`);
+const alias = recipientAddresses.map((_, i) => `chain2Recipient${i + 1}`);
 const fundAmount = ethers.utils.parseUnits("1000", 6);
 const sendAmount = ethers.utils.parseUnits("100", 6);
 
 (async () => {
   console.log("==== Preparing chain1... ====");
   const chain1 = await axelar.createNetwork({ seed: "chain1" });
-  const [sender] = chain1.userWallets;
+  const [chain1Sender] = chain1.userWallets;
   console.log("\n==== Preparing chain2... ====");
   const chain2 = await axelar.createNetwork({ seed: "chain2" });
-  const [deployer] = chain2.userWallets;
+  const [chain2Deployer] = chain2.userWallets;
 
   // Deploy DistributionExecutor contract
   console.log("\n==== Deploying DistributionExecutor contract... ====");
   const distributionContract = await deployContract(
-    deployer,
+    chain2Deployer,
     distributionExecutorAbi,
     [chain2.gateway.address]
   );
   console.log("Deployed:", distributionContract.address);
 
   // Fund sender account with 1000 UST
-  await chain1.giveToken(sender.address, "UST", fundAmount);
+  await chain1.giveToken(chain1Sender.address, "UST", fundAmount);
 
   console.log("\n==== Initial balances ====");
-  await printBalance("Sender", sender.address, chain1.ust);
+  await printBalance("chain1Sender", chain1Sender.address, chain1.ust);
   await printMultipleBalances(alias, recipientAddresses, chain2.ust);
 
   // Approve the AxelarGateway to use our UST on chain1.
   await (
-    await chain1.ust.connect(sender).approve(chain1.gateway.address, sendAmount)
+    await chain1.ust.connect(chain1Sender).approve(chain1.gateway.address, sendAmount)
   ).wait();
 
   console.log("\n==== Calling the gateway contract ====");
@@ -50,7 +50,7 @@ const sendAmount = ethers.utils.parseUnits("100", 6);
     [recipientAddresses]
   );
   const tx = await chain1.gateway
-    .connect(sender)
+    .connect(chain1Sender)
     .callContractWithToken(
       chain2.name,
       distributionContract.address,
@@ -66,6 +66,6 @@ const sendAmount = ethers.utils.parseUnits("100", 6);
   await axelar.relay();
 
   console.log("\n==== After cross-chain balances ====");
-  await printBalance("Sender", sender.address, chain1.ust);
+  await printBalance("chain1Sender", chain1Sender.address, chain1.ust);
   await printMultipleBalances(alias, recipientAddresses, chain2.ust);
 })();

--- a/src/send-token.js
+++ b/src/send-token.js
@@ -1,7 +1,6 @@
 const axelar = require("@axelar-network/axelar-local-dev");
 const { ethers } = require("ethers");
 const { printBalance } = require("./utils/logger");
-const { AxelarGateway } = require("@axelar-network/axelarjs-sdk");
 
 const fundAmount = ethers.utils.parseUnits("1000", 6);
 const sendAmount = ethers.utils.parseUnits("100", 6);

--- a/src/send-token.js
+++ b/src/send-token.js
@@ -8,28 +8,28 @@ const sendAmount = ethers.utils.parseUnits("100", 6);
 (async () => {
   console.log("==== Preparing chain1... ====");
   const chain1 = await axelar.createNetwork();
-  const [user1] = chain1.userWallets;
+  const [chain1User] = chain1.userWallets;
   console.log("\n==== Preparing chain2... ====");
   const chain2 = await axelar.createNetwork();
-  const [user2] = chain2.userWallets;
+  const [chain2User] = chain2.userWallets;
 
-  await chain1.giveToken(user1.address, "UST", fundAmount);
+  await chain1.giveToken(chain1User.address, "UST", fundAmount);
 
   console.log("\n==== Initial balances ====");
-  await printBalance("user1", user1.address, chain1.ust);
-  await printBalance("user2", user2.address, chain2.ust);
+  await printBalance("chain1User", chain1User.address, chain1.ust);
+  await printBalance("chain2User", chain2User.address, chain2.ust);
 
   // Approve the AxelarGateway to use our UST on chain1.
   await (
-    await chain1.ust.connect(user1).approve(chain1.gateway.address, sendAmount)
+    await chain1.ust.connect(chain1User).approve(chain1.gateway.address, sendAmount)
   ).wait();
 
   // Send it to the gateway contract
   console.log("\n==== Send token to the gateway contract ====");
   const receipt = await (
     await chain1.gateway
-      .connect(user1)
-      .sendToken(chain2.name, user2.address, "UST", sendAmount)
+      .connect(chain1User)
+      .sendToken(chain2.name, chain2User.address, "UST", sendAmount)
   ).wait();
   console.log("Tx", receipt.transactionHash);
 
@@ -37,6 +37,6 @@ const sendAmount = ethers.utils.parseUnits("100", 6);
   await axelar.relay();
 
   console.log("\n==== After cross-chain balances ====");
-  await printBalance("user1", user1.address, chain1.ust);
-  await printBalance("user2", user2.address, chain2.ust);
+  await printBalance("chain1User", chain1User.address, chain1.ust);
+  await printBalance("chain2User", chain2User.address, chain2.ust);
 })();

